src/main/configurations/Translate/Configuration_GenereerIdentificatieEmulator.xml

```diff
@@ -11,7 +11,7 @@
                 slotId="${instance.name}/genereerIdentificatieEmulator"/>
         </Receiver>
 
-        <Pipeline>
+        <Pipeline transactionAttribute="RequiresNew">
```

Comment:
Why RequiresNew and not Required?
And maybe set the transactionAttribute on the Receiver.
Not really related to this PR, but why is there a JdbcErrorStorage on this adapter? In case this adapter fails I don't think it has any use to manually retry it from the error store. The result of a manual rerun will never be used.

Reply:
Changed to Required instead of RequiresNew. I re-read the documentation and I appear to have misunderstood the definition.

Relocated TransactionAttribute to the receiver as the Frank!Manual recommends this practice as well.

The errorStore was not my work, but you discussed this with Marcel during today's meeting.